docs/content/migration/v2.md

```diff
@@ -560,3 +560,19 @@ To enable these ciphers, please set the option `CipherSuites` in your [TLS confi
 > (https://go.dev/doc/go1.22#crypto/tls)
 
 To enable TLS 1.0, please set the option `MinVersion` to `VersionTLS10` in your [TLS configuration](https://doc.traefik.io/traefik/https/tls/#cipher-suites) or set the environment variable `GODEBUG=tls10server=1`.
+
+### <EntryPoint>.Transport.RespondingTimeouts.ReadTimeout
+
+Starting with `v2.11.1` the entryPoints `readTimeout` option default value changed to 5 seconds.
+For HTTP, this option defines the maximum duration for reading the entire request, including the body.
+For TCP, this option defines the maximum duration for all read operations on the connection.
+
+The default value was previously set to zero, which means no timeout.
+This change has been done to avoid Traefik instances with default configuration to be hanging forever while waiting for bytes to be read on the connection.
```

Comment:
```suggestion

This change has been done to avoid Traefik instances with the default configuration to be hanging forever while waiting for bytes to be read on the connection.
```